Including language around UDR support for V2 SKU (#32376)
Azure/azure-content: articles/application-gateway/configuration-overview.md

For the v1 SKU, user-defined routes (UDRs) are supported on the Application Gateway subnet, as long as they don't alter end-to-end request/response communication. For example, you can set up a UDR in the Application Gateway subnet to point to a firewall appliance for packet inspection. But you must make sure that the packet can reach its intended destination after inspection. Failure to do so might result in incorrect health-probe or traffic-routing behavior. This includes learned routes or default 0.0.0.0/0 routes that are propagated by Azure ExpressRoute or VPN gateways in the virtual network.

For the v2 SKU, UDRs are not supported on the Application Gateway subnet. For more information, see [Azure Application Gateway v2 SKU](application-gateway-autoscaling-zone-redundant.md#differences-with-v1-sku).

> [!NOTE]

> > UDRs will not be supported for the v2 SKU.  If you require UDRs you should continue to deploy v1 SKU.

Reviewer: ```suggestion
> UDRs are not supported for the v2 SKU.  If you require UDRs you should continue to deploy v1 SKU.
```

> > UDRs will not be supported for the v2 SKU at General Availability.  If you require UDRs you should continue to deploy v1 SKU.

Reviewer: ```suggestion
> UDRs aren't supported for the v2 SKU.  If you require UDRs, you should continue to deploy the v1 SKU.
```

> > UDRs will not be supported for the v2 SKU at General Availability.  If you require UDRs you should continue to deploy v1 SKU.

Reviewer: @johnwildes
v2 is now post GA. Please remove the reference to GA and just state that UDRs aren't currently supported.

Author: I've updated the line.
